src/applications/mhv/landing-page/utilities/data/index.js

```diff
@@ -71,19 +74,31 @@ const resolveLandingPageLinks = (authdWithSSOe = false, featureToggles) => {
         href: null,
         oldHref: mhvUrl(authdWithSSOe, 'prescription_refill'),
         text: 'Refill VA prescriptions',
-        toggle: null,
+        toggle: FEATURE_FLAG_NAMES.mhvMedicationsToVaGovRelease,
       },
       {
         href: null,
         oldHref: mhvUrl(authdWithSSOe, '/prescription-tracking'),
         text: 'Track prescription delivery',
-        toggle: null,
+        toggle: FEATURE_FLAG_NAMES.mhvMedicationsToVaGovRelease,
       },
       {
         href: null,
         oldHref: mhvUrl(authdWithSSOe, '/my-complete-medications-list'),
         text: 'Medications and allergies',
-        toggle: null,
+        toggle: FEATURE_FLAG_NAMES.mhvMedicationsToVaGovRelease,
+      },
```

Comment:
These three Medications links will go away when `FEATURE_FLAG_NAMES.mhvMedicationsToVaGovRelease` is true.